src/main/java/de/bonndan/nivio/input/LabelToFieldProcessor.java

```diff
@@ -79,9 +81,15 @@ private void setValue(ItemDescription item, String name, String value) {
                 return;
             }
 
-            myAccessor.setPropertyValue(name, value.trim());
+            if (name.startsWith(Linked.LINK_LABEL_PREFIX)) {
+                item.setLink(name.replace(Linked.LINK_LABEL_PREFIX, ""), new URL(value));
+            } else {
+                myAccessor.setPropertyValue(name, value.trim());
+            }
         } catch (NotWritablePropertyException e) {
             logger.warn("Failed to write field '" + name + "' via label");
+        } catch (MalformedURLException e) {
+            logger.warn("Failed to add link '" + name + "' via label because of malformed URL " + value);
```

Comment:
You can pass the exception as last parameter to ensure that it is not completely lost.

Reply:
This is nivio's special Logger decorator, meaning everything logged here lands in the GUI